File: day seven/oldSnacks/CheckOutSystem.java
import java.util.Date;
import java.util.Scanner;
import java.util.ArrayList;


	public class CheckOutSystem{
	
		public static void main(String[]args){

		Scanner input = new Scanner(System.in);
		
		int counter = 0;
		double total = 1;
		double subTotal = 0;


		ArrayList<String> items = new ArrayList<>();
		ArrayList<Double> quantity = new ArrayList<>();
		ArrayList<Double> price = new ArrayList<>();
		ArrayList<Double> totals = new ArrayList<>();

		
		System.out.println("What is the customer's name: ?");
		String customerName = input.nextLine();
			
		String userResponse = "";
			
		Date date = new Date();

		do{		
			

			System.out.println("What did the user buy: ?");
			String itemBought = input.next();
			
			System.out.println("How many pieces of goods: ?");
			double numberOfPieces  = input.nextInt();

			System.out.println("How much per unit: ?");
			double pricePerUnit = input. nextInt();

			System.out.println("Add more items: ?");
			userResponse = input.next();
			
			total = numberOfPieces * pricePerUnit;
			items.add(itemBought);
			quantity.add(numberOfPieces);
			price.add(pricePerUnit);
			totals.add(total);

		}while(!userResponse.equals("no"));
			
		System.out.println("What is your name:?");
		String cashierName = input.next();
		System.out.println("How much discount will he get: ");
		int discountGiven = input.nextInt();
		
			System.out.println("SEMICOLON STORES ");
			System.out.println("MAIN BRANCH ");
			System.out.println("LOCATION :  312, HERBERT MACAULAY WAY, SABO YABO, LAGOS");
			System.out.println("TEL: 03293828343 ");
			System.out.println(date);
			System.out.println("Cashier:  " + cashierName);
			System.out.println("Customer's Name:  " + customerName);

		System.out.println("=================================");
		System.out.println("ITEM \t QTY \t PRICE \t  TOTAL");
		System.out.println("----------------------------------");
		for(counter = 0; counter < items.size(); counter++){
		System.out.println(items.get(counter) + "\t" + quantity.get(counter) + "\t" + price.get(counter) + "\t" + totals.get(counter));
	subTotal += totals.get(counter);
      }

double discount = discountGiven * subTotal / 100;
double vat = 17.50 / 100 * subTotal;
double BillTotal = subTotal + discount + vat;
System.out.println("--------------------------------------");	

System.out.println( "\t" + "Sub Total:" + "\t" +  subTotal);	
System.out.println( "\t" + "Discount:" + "\t" +   discount);
System.out.printf(" 	VAT  @17.50:   %2f%n", vat);
		
System.out.println("--------------------------------------");	

System.out.printf("	BillTotal:     %.2f%n", BillTotal); 

System.out.println(" ======================================");
System.out.println("THIS IS NOT A RECEIPT, KINDLY PAY " + BillTotal);
System.out.println("=========================================");
	
System.out.println("How much did the customer give to you? ");
double AmountPaid = input.nextDouble();

	System.out.println("SEMICOLON STORES ");
			System.out.println("MAIN BRANCH ");
			System.out.println("LOCATION :  312, HERBERT MACAULAY WAY, SABO YABO, LAGOS");
			System.out.println("TEL: 03293828343 ");
			System.out.println(date);
			System.out.println("Cashier:  " + cashierName);
			System.out.println("Customer's Name:  " + customerName);

	System.out.println("=================================");
	System.out.println("ITEM \t QTY \t PRICE \t  TOTAL");
	System.out.println("----------------------------------");
	
 discount = discountGiven * subTotal / 100;
 vat = 17.50 / 100 * subTotal;
 BillTotal = subTotal + discount + vat;
  double Balance = BillTotal - AmountPaid;
System.out.println("--------------------------------------");	

System.out.println( "\t" + "Sub Total:" + "\t" +  subTotal);	
System.out.println( "\t" + "Discount:" + "\t" +   discount);
System.out.printf(" 	VAT  @17.50:   %2f%n", vat);
		
System.out.println("--------------------------------------");	

System.out.printf("	BillTotal:    %.2f%n",   BillTotal); 
System.out.printf("      AmountPaid:    %.2f%n ",  AmountPaid);
System.out.printf("	 Balance:     %.2f%n",   Balance);
System.out.println("===========================================");
       System.out.println("THANK YOU FOR YOUR PATRONAGE ");
System.out.println("========================================");

	}

}

File: day seven/oldSnacks/checkoutsystem.py
from datetime import datetime

counter = 0
total = 1
sub_total = 0
quantity = []
items = []
price = []
grand_total = []

print("WELCOME TO SEMICOLON STORES")

customer_name = input("\nWhat is the customer's Name? ")

user_response = ""

today = datetime.today()

while user_response != "no":
	item_bought = input("What did the user buy? ")
	number_of_pieces = int(input("How many pieces of goods? "))
	price_per_unit = int(input("How much per unit? "))
	user_response = input("Add more items? (yes/no) ")

	total = number_of_pieces * price_per_unit

	cashier_name = input("What is your name? ")

	discount_given = int(input("How much discount will they get? "))

	print("SEMICOLON STORES")

	print("MAIN BRANCH")

	print("LOCATION: 312, HERBERT MACAULAY WAY, SABO YABO, LAGOS")

	print("TEL: 03293828343")

	print(today)

	print("Cashier:", cashier_name)

	print("Customer's Name:", customer_name)

	print("=================================")

	print("ITEM \t QTY \t PRICE \t  TOTAL")

	print("----------------------------------")

	for counter in range(len(items)):
        	print(items[counter], "\t", quantity[counter], "\t", price[counter], "\t", totals[counter])
        	sub_total += totals[counter]

	discount = discount_given * sub_total / 100

	vat = 17.50 / 100 * sub_total

	bill_total = sub_total + discount + vat

	print("--------------------------------------")

	print("\tSub Total:\t", sub_total)

	print("\tDiscount:\t", discount)

	print("\tVAT @17.50:\t", vat)

	print("--------------------------------------")

	print("\tBill Total:\t", bill_total)

	print("=================================")

	print("THIS IS NOT A RECEIPT, KINDLY PAY", bill_total)

	print("=================================")

	amount_paid = float(input("How much did the customer give to you? "))

	balance = bill_total - amount_paid

	print("SEMICOLON STORES")

	print("MAIN BRANCH")

	print("LOCATION: 312, HERBERT MACAULAY WAY, SABO YABO, LAGOS")

	print("TEL: 03293828343")

	print(today)

	print("Cashier:", cashier_name)

	print("Customer's Name:", customer_name)

	print("=================================")

	print("ITEM \t QTY \t PRICE \t  TOTAL")

	print("----------------------------------")

	print("--------------------------------------")

	print("\tSub Total:\t", sub_total)

	print("\tDiscount:\t", discount)

	print("\tVAT @17.50:\t", vat)

	print("--------------------------------------")

	print("\tBill Total:\t", bill_total)

	print("\tAmount Paid:\t", amount_paid)

	print("\tBalance:\t", balance)

	print("=================================")

	print("THANK YOU FOR YOUR PATRONAGE")

	print("=================================")

	

	

	

	

	

	



	

